src/LecturaArchivoConfiguracion.cpp

```diff
@@ -100,6 +100,8 @@ void LecturaArchivoConfiguracion::crearCelula(std::string tableroId, int fila, i
 	float tasaNacimiento = parcelaAsociada.getfactorNacimiento();
 	Celula * celulaASetear = parcelaAsociada.getCelula();
 	celulaASetear->nacer(tasaNacimiento, &rgbCelula);
+	tablero->getDatosTablero()->sumarCelulaViva();
+	tablero->getDatosTablero()->reiniciarContadorNacidasMuertasEnUltimoTurno(); //Para que no las cuente como nacidas en primer turno
```

Comment:
Lo llamaria reiniciarContadorDeNacidasYMuertasEnUltimoTurno, suena raro NacidasMuertas jaja.
Por otro lado por qué al crear las celulas a partir del archivo no estaría esos contadores en cero?

Reply:
Ahora le cambio el nombre. 
Por que el metodo sumarCelulaViva(); hace:
	this->nacidasEnUltimoTurno ++;
	this->cantidadCelulasVivas ++;
	this->nacidasTotal ++;

Reply:
:+1: